scripts/run_notebooks/runner.py

```diff
@@ -2,21 +2,41 @@
 
 import logging
 from pathlib import Path
+from tempfile import NamedTemporaryFile
+from uuid import uuid4
 
 import papermill
 from joblib import Parallel, delayed
-from tqdm import tqdm
+from nbformat.notebooknode import NotebookNode
+from papermill.iorw import load_notebook_node, write_ipynb
+
+HERE = Path(__file__).parent
 
 KERNEL_NAME: str = "python3"
 DOC_SOURCE = Path("docs/source")
 NOTEBOOKS_PATH = DOC_SOURCE / "notebooks"
+FULL_RUNS = [
+    # Samples from HalfFlat distribution
+    # "clv_quickstart.ipynb",
+    # "other_nuts_samplers.ipynb",
+    # "sBG.ipynb",
+    # Uses the MAP estimate
+    # "bg_nbd.ipynb",
+    # "gamma_gamma.ipynb",
+    # "pareto_nbd.ipynb",
+]
 NOTEBOOKS_SKIP: list[str] = [
-    "mmm_tvp_example.ipynb",  # This notebook takes too long to run
+    # This notebook takes too long to run
+    # "other_nuts_samplers.ipynb",
```

Comment:
@wd60622 So if I understand this correctly, you will uncomment these notebook lists before merging as these lists decide full runs and mock runs?

Reply:
Why does "other_nuts_samplers.ipynb" appear in FULL RUNS and NOTEBOOKS_SKIP?

Reply:
I can actually remove all of the comments now. We can leave empty lists or populate if we want to either skip or run without a mock. 

Should I leave empty without comments or remove entirely?

Reply:
I think we can remove entirely if we are not using it